### DISCUSSION TITLE
Can I join your team

### DISCUSSION THREAD

**Discussion by onkarsinghsahil** [ROM-builders/temporary#11323] [Q&A]
Voltage successful build
https://cirrus-ci.com/build/6229490501156864

Local manifest: https://github.com/onkarsinghsahil/local_manifest/blob/voltage/local_manifest.xml

Device tree:
https://github.com/onkarsinghsahil/device_xiaomi_mido/commits/voltage

Vendor tree:
https://github.com/onkarsinghsahil/proprietary_vendor_xiaomi

Kernal tree:
https://github.com/onkarsinghsahil/kernel_mido

Telegram I'd: @error7951

**Comment by Apon77** [comment]
id mismatch

**Reply by onkarsinghsahil** [reply]
@error7951

**Reply by onkarsinghsahil** [reply]
Which one sir telegram id

**Reply by onkarsinghsahil** [reply]
https://t.me/error7951

**Reply by Apon77** [reply]
check telegram pm

> https://t.me/error7951
